extras/docker/Dockerfile

```diff
@@ -7,23 +7,23 @@ RUN apt-get update && apt-get install -y git cmake golang && \
 
 RUN git clone --recurse-submodules --depth 1 https://github.com/cloudflare/quiche
 RUN cd quiche && \
-    cargo build --release --examples
-RUN cd quiche && \
-    cargo build --manifest-path tools/apps/Cargo.toml --release
+    git log --oneline -n 1 && \
+    cargo build --examples && \
+    cargo build --manifest-path tools/apps/Cargo.toml
 
 ##
 ## quiche-base: base quiche image
 ##
 FROM debian:latest as quiche-base
+
 RUN apt-get update && apt-get install -y ca-certificates && \
     rm -rf /var/lib/apt/lists/*
-RUN update-ca-certificates
```

Comment:
did you mean to remove `RUN update-ca-certificates`? 

The (perhaps mistaken) intent was to ensure the image's CA store was up to date to trust public sites. If that is not required, then there seems no need to install `ca-certificates` package above.

Reply:
- `update-ca-certificates` will run when you install the package, so no need to run it twice.
- Also in quiche image, `ca-certificates` is not installed by default, it you need to give `--no-verify` for public websites (e.g. blog.cloudflare.com) every time, so I keep it.

Reply:
makes sense